code-challenges.test.js

```diff
@@ -14,38 +16,114 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
+describe ("whoAreU", () => {
+  it ("Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+expect (whoAreU(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+
+  })
+})
 
 // b) Create the function that makes the test pass.
 
+// Create a function name whoAreU
+// Create a variable name capthat, capThat will return the updated values of the names in people 
+// Use .map to iterate through people 
+// Use .toLowerCase to lower case all the letters in name: 
+// Use .split to separate string in name:
+// Use .map to iterate the words in name:, .chartAt index [0] .toUpperCase the letter in each string in name:
+// Use .substring index [1] to stop the .toUpperCase from capitalizing the whole word, then use the .join to bring the string together 
+// Return capThat with `is ${value.occupation} to add the occupation:
+
+function whoAreU (array) {
+  return array.map(value => {
+    var capThat = (value.name).toLowerCase().split(' ').map(word => word.charAt(0).toUpperCase() + word.substring(1)).join(' ')
+    return capThat + " " + `is ${value.occupation}.`
```

Comment:
It looks like you are using a combination of string concatenation and string interpolation.  I would recommend using one or the other as this is best practice.